src/components/organisms/ChatSidebar/components/PrivateChats/PrivateChats.tsx

```diff
@@ -34,6 +34,10 @@ export const PrivateChats: React.FC<PrivateChatsProps> = ({
   const { selectRecipientChat } = useChatSidebarControls();
 
   const numberOfOnline = onlineUsers.length;
+  const privateChatUsers: String[] = [];
+  privateChatPreviews.forEach((user) =>
+    privateChatUsers.push(user.counterPartyUser.id)
```

Comment:
`.map` creates a new array. Use it instead of `forEach` and `push`

```tsx
const privateChatUsers = privateChatPreviews.map(chatPreview => chatPreview.counterPartyUser.id)
```

Reply:
I also suggest renaming `privateChatUsers` into for example `contactedChatUsers`

Reply:
+1 to both @denisdimitrov and @mike-lvov 's feedback here

Reply:
> I also suggest renaming privateChatUsers into for example contactedChatUsers

Eh.. maybe. But if you're going to do that, then you would probably also want/need to change `privateChatPreviews` to some more meaningful/useful name. Feels a little irrelevant/OOS to me. If anything, I feel like `privateChatUsers` is potentially a good name, and `privateChatPreviews` could change. But at the end of the day, not sure that either need to.